corehq/apps/users/static/users/js/web_users.js

```diff
@@ -73,6 +76,125 @@ hqDefine("users/js/web_users",[
         $("#web-users-panel").koApplyBindings(webUsersList());
     });
 
+    /* Invitations panel */
+    var Invitation = function (options) {
+        assertProperties.assertRequired(options, ["uuid", "email", "email_marked_as_bounced", "invited_on", "role_label"]);
+        var self = _.extend({}, options);
+        self.invited_on = ko.observable(new Date(self.invited_on));
+        self.invitedOnText = ko.computed(function () {
+            return moment(self.invited_on()).format("MMMM Do YYYY, h:mm a");
```

Comment:
Does this deal with translations? Otherwise this says "July 15th 2020", right? I'm not sure how we've dealt with that in the past.

Reply:
It doesn't. It does look like moment has locale support and we use it in the ICDS dashboard but not anywhere else we use moment (which I think is mostly web apps). Added a trello card: https://trello.com/c/5YDsdYoT/109-translations-for-momentjs